webpack/components/ReportJsonViewer.js

```diff
@@ -10,7 +10,17 @@ const theme = {
 
 const ReportJsonViewer = ({ data }) => (
   <div className="report-json-viewer">
-    <JSONTree data={data} hideRoot theme={theme} />
+    <JSONTree
+      data={data}
+      hideRoot
+      theme={theme}
+      shouldExpandNode={(keyPath, myData, level) =>
```

Comment:
+1 for the catch with `level === 3`.

Code can be slightly simplified:
```js
      shouldExpandNode={(keyPath, _myData, level) =>
        keyPath[0] === '_meta' ||
        keyPath[0] === 'hostvars' ||
        level === 3 ||
        keyPath[0] === 'foreman_ansible_roles'
      }
```
Chances that there will be `_meta`, `hostvars` or `foreman_ansible_roles` on other levels are pretty slim,
so we can skip the level condition IMHO. What do you think?

The underscore in `_myData` tells developer that this variable is not used in the function, it's just there doing nothing :D